api/prisma/schema.prisma

```diff
@@ -15,21 +15,29 @@ enum Role {
   ADMIN
 }
 
+enum Sex {
+  MALE
+  FEMALE
+  UNSPECIFIED
```

Comment:
```suggestion
  UNSPECIFIED
  DIVERS
```

Reply:
`UNSPECIFIED` soll Divers darstellen. Für "Nicht angegeben" kann in der Datenbank einfach `NULL` stehen.

Reply:
Ich denke wir sollten das dann umbennen zu DIVERS das ist explizitier und politisch vielleicht auch besser

Reply:
Das englische Pendant wäre **various** oder eben **unspecified**. Einen Mix aus Deutsch und Englisch halte ich für kontraproduktiv.

Reply:
`UNSPECIFIED` ist hier IMHO irreführend. Als diverse Person ist man ja nicht geschlechtlich unspezifiziert, sondern eben gerade doch spezifiziert, nur eben als nicht-männlich und nicht-weiblich.

Die englische Entsprechung des Geschlechtseintrags "divers" ist "non-binary", wobei die Antidiskriminierungsstelle des Bunds in ihren englischsprachigen Veröffentlichungen[0] auch die Begriff "diverse" und "third option" verwendet. Also vielleicht `DIVERSE` mit "E"?

0: https://www.antidiskriminierungsstelle.de/EN/about-discrimination/grounds-for-discrimination/gender-and-gender-identity/third-option/third-option-node.html

Reply:
Ich sehe gerade, dass  das Feld mittlerweile zu `GENDER` umbenannt wurde. IIRC wird das Geschlechtsattribut für die Zeltbelegung benötigt. Ich vermute, dass `SEX` dann hier der korrekte Begriff wäre, oder @danielswiatek? Insbesondere beschreibt Intergeschlechtlichkeit[0] ("divers" bzw. "third option") ein biologisches Geschlecht ("Sex") und keine soziale Geschlechtsidentität ("Gender").

1: https://de.wikipedia.org/wiki/Intergeschlechtlichkeit